io.catenax.vehicle.product_description/3.0.0/ProductDescription.ttl

```diff
@@ -0,0 +1,511 @@
+   samm:characteristic :FuelKBA ;
+   samm:exampleValue "Diesel" .
+
+:nhtsaFuelType a samm:Property ;
+   samm:preferredName "Fuel type(NHTSA)"@en ;
+   samm:description "description of the fuel according US NHTSA"@en ;
+   samm:characteristic :FuelNHTSA ;
+   samm:exampleValue "Diesel" .
+
+:OemShortNameTrait a samm-c:Trait ;
+   samm-c:baseCharacteristic :WorldManufacturerInformation ;
+   samm-c:constraint :WorldManufacturerInformationCodeLength .
+
+:PositiveNumber a samm:Characteristic ;
+   samm:dataType xsd:positiveInteger .
+
+:KbaVariant a samm-c:Enumeration ;
+   samm:preferredName "KBA Variant"@en ;
+   samm:description "Current version of the Enumeration is sub-set of list defined from the German Federal Office for motor vehicles. "@en ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "Limousine" "Schr?ghecklimousine" "Kombilimousine" "Coup?" "Kabrio-Limousine" "Cabrio-Limousine" "Mehrzweckfahrzeug" "Pkw-Pick-up" "Van" "Pick-up" ) .
+
+:NhtsaVariant a samm-c:Enumeration ;
+   samm:preferredName "NHTSA Variant"@en ;
+   samm:description "Enumeration comming from NHTSA offline database vpic, table [vPICList_lite].[dbo].[BodyStyle]"@en ;
+   samm:see <https://vpic.nhtsa.dot.gov/api/> ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "Cargo Van" "Convertible" "Cabriolet" "Coupe" "Crossover Utility Vehicle(CUV)" "Hatchback" "Liftback" "Notchback" "Limousine" "Low Speed Vehicle(LSV)" "Neighborhood Electric Vehicle(NEV)" "Minivan" "Pickup" "Roadster" "Sedan" "Saloon" "Sport Utility Truck(SUT)" "Sport Utility Vehicle(SUV)" "Multi-Purpose Vehicle(MPV)" "Van" "Wagon" ) .
+
+:CountryCodeTrait a samm-c:Trait ;
+   samm-c:baseCharacteristic :CountryCodeCharacteristic ;
+   samm-c:constraint :CountryCodeRegularExpression .
+
+:CubicCapacity a samm-c:Measurement ;
+   samm:preferredName "Cubic Capacity"@en ;
+   samm:description "cubic capacity of the engine"@en ;
+   samm:dataType xsd:integer ;
+   samm-c:unit unit:cubicCentimetre .
+
+:FuelKBA a samm-c:Enumeration ;
+   samm:preferredName "Fuel type (KBA)"@en ;
+   samm:description "enumeration of possible fuel types of a vehicle according german KBA"@en ;
+   samm:see <https://www.kba.de/SharedDocs/Downloads/DE/SV/sv221_m1_schad_pdf.pdf> ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "Unbekannt" "Diesel" "Benzin" "Vielstoff" "Elektro" "Fl?ssiggas" "Benzin/Fl?ssiggas" "Benzin/komp.Erdgas" "Hybr.Benzin/E" "Erdgas NG" "Hybr.Diesel/E" "Wasserstoff" "Hybr.Wasserst./E" "Wasserstoff/Benzin" "Wasserst./Benzin/E" "BZ/Wasserstoff" "BZ/Benzin" "BZ/Methanol" "BZ/Ethanol" "Hybr.Vielstoff/E" "Methan" "Benzin/Methan" "Hybr.Erdgas/E" "Benzin/Ethanol" "Hybr.Fl?ssiggas/E" "Hybr.B/E ext.aufl." "Hybr.D/E ext.aufl." "Hybr.LPG/E ext.aufl." "Hybr.W/E ext.aufl." "Hybr.V/E ext.aufl." "Hybr.NG/E ext.aufl." "Hybr.Wod.B/Eext.aufl" "Wasserstoff/NG" "Hybr.W/NG/E ext.aufl" "Ethanol" "Hybr.BZ/W/E" "Hybr.BZ/W/E ext. aufl." "Zweistoff LNG/Diesel" "Verfl?ssigtes Erdgas (LNG)" "Andere" ) .
```

Comment:
Vergl?sigtes Erdgas -> Encoding Problem?

Reply:
Is fixed - please recheck